xray/segment.go

```diff
@@ -271,8 +286,23 @@ func NewSegmentFromHeader(ctx context.Context, name string, r *http.Request, h *
 	return con, seg
 }
 
+// Check if SDK is disabled
+func Disabled() bool {
```

Comment:
Especially if removing the local variable, probably good to name this `func SdkDisabled()`

Reply:
right.